Index: lib/Makefile
===================================================================
--- lib/Makefile	(revision 305192)
+++ lib/Makefile	(working copy)
@@ -60,7 +60,7 @@
 	${_libgssapi} \
 	${_librpcsec_gss} \
 	${_libiconv_modules} \
-	libifc \
+	libifconfig \
 	libipsec \
 	libjail \
 	libkiconv \
Index: share/mk/bsd.libnames.mk
===================================================================
--- share/mk/bsd.libnames.mk	(revision 305192)
+++ share/mk/bsd.libnames.mk	(working copy)
@@ -82,7 +82,7 @@
 LIBIBSDP?=	${DESTDIR}${LIBDIR}/libibsdp.a
 LIBIBUMAD?=	${DESTDIR}${LIBDIR}/libibumad.a
 LIBIBVERBS?=	${DESTDIR}${LIBDIR}/libibverbs.a
-LIBIFC?=	${DESTDIR}${LIBDIR}/libifc.a
+LIBIFCONFIG?=	${DESTDIR}${LIBDIR}/libifconfig.a
 LIBIPSEC?=	${DESTDIR}${LIBDIR}/libipsec.a
 LIBJAIL?=	${DESTDIR}${LIBDIR}/libjail.a
 LIBKADM5CLNT?=	${DESTDIR}${LIBDIR}/libkadm5clnt.a
Index: share/mk/src.libnames.mk
===================================================================
--- share/mk/src.libnames.mk	(revision 305192)
+++ share/mk/src.libnames.mk	(working copy)
@@ -105,7 +105,7 @@
 		heimntlm \
 		heimsqlite \
 		hx509 \
-		ifc \
+		ifconfig \
 		ipsec \
 		jail \
 		kadm5clnt \
