Index: share/mk/bsd.libnames.mk
===================================================================
--- share/mk/bsd.libnames.mk	(revision 304215)
+++ share/mk/bsd.libnames.mk	(working copy)
@@ -82,6 +82,7 @@
 LIBIBSDP?=	${DESTDIR}${LIBDIR}/libibsdp.a
 LIBIBUMAD?=	${DESTDIR}${LIBDIR}/libibumad.a
 LIBIBVERBS?=	${DESTDIR}${LIBDIR}/libibverbs.a
+LIBIFCONFIG?=	${DESTDIR}${LIBDIR}/libifconfig.a
 LIBIPSEC?=	${DESTDIR}${LIBDIR}/libipsec.a
 LIBJAIL?=	${DESTDIR}${LIBDIR}/libjail.a
 LIBKADM5CLNT?=	${DESTDIR}${LIBDIR}/libkadm5clnt.a
Index: share/mk/src.libnames.mk
===================================================================
--- share/mk/src.libnames.mk	(revision 304215)
+++ share/mk/src.libnames.mk	(working copy)
@@ -105,6 +105,7 @@
 		heimntlm \
 		heimsqlite \
 		hx509 \
+		ifconfig \
 		ipsec \
 		jail \
 		kadm5clnt \
